docs/notes.md

```diff
@@ -66,13 +66,13 @@ instead of \\(\operatorname{Com}(v, {\widetilde{v}})\\) for brevity.
 We also make use of *vector Pedersen commitments*, which we define for
 pairs of vectors as \\[
 \begin{aligned}
-    \operatorname{Com}({\mathbf{a}}\_{L}, {\mathbf{a}}\_{R}) 
- &= \operatorname{Com}({\mathbf{a}}\_{L}, {\mathbf{a}}\_{R}, {\widetilde{a}})
-  = {\langle {\mathbf{a}}\_{L}, {\mathbf{G}} \rangle} + {\langle {\mathbf{a}}\_{R}, {\mathbf{H}} \rangle} + {\widetilde{a}} {\widetilde{B}},\end{aligned}
+    \operatorname{Com}({\mathbf{a}}\_{\operatorname{lo}}, {\mathbf{a}}\_{\operatorname{hi}}) 
```

Comment:
These `a_L` and `a_R` are different vectors (left and right bit commitments), not the low and high halves of a single vector...

Reply:
Ah, oops. Will fix...